app/AvailablePlugin/PasswordAuthenticator/Lib/lang.php

```diff
@@ -86,15 +86,24 @@
   'pl.passwordauthenticator.ssr.for'        => 'Select a new password for %1$s.',
   'pl.passwordauthenticator.ssr.hr.sent'    => 'Password reset request sent to "%1$s"',
   'pl.passwordauthenticator.ssr.info'       => 'Enter a verified email address or registered identifier to proceed.</p>If you still know your password, click <a href="%1$s">here</a> to directly select a new password.',
+  'pl.passwordauthenticator.ssr.mt'         => 'Reset Message Template',
   'pl.passwordauthenticator.ssr.mt.desc'    => 'Message template used for email to send reset instructions to',
-  'pl.passwordauthenticator.ssr.q'          => 'Email Address or Identifier',
   'pl.passwordauthenticator.ssr.redirect'   => 'Redirect on Self Service Reset',
   'pl.passwordauthenticator.ssr.redirect.desc' => 'URL to redirect to on successful self service reset',
-  'pl.passwordauthenticator.ssr.sent'       => 'An email with further instructions has been sent to the address on record',
   'pl.passwordauthenticator.ssr.url'        => 'Self Service Reset Initiation URL',
   'pl.passwordauthenticator.ssr.validity'   => 'Self Service Reset Token Validity',
   'pl.passwordauthenticator.ssr.validity.desc' => 'Time in minutes the reset token is valid for',
+  'pl.passwordauthenticator.ssrorreminder.q' => 'Email Address or Identifier',
```

Comment:
Ok, should I add an identical key for .reminder? Or should I reuse the ssr keys (even though they would be used for remind)? I'm not aware of all the implications here.

Reply:
It's the same text for the same context, we don't need two keys, especially since it's in a plugin so if we do need to break the keys apart later for some reason it's not a big deal.